CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # NHS.UK prototype kit Changelog
 
+## Unreleased
+
+:new: **New features**
+
+- Update to NHS.UK frontend v9.0.0
+- Drop `outerContent` block and use `beforeContent` block for both breadcrumbs and back links
```

Comment:
This could possibly have more narrative as users may need to update their layouts?

Reply:
Or else link to the release notes in NHS.UK frontend that tell them what to do.

Reply:
Good idea. Although we might also want to put that narrative on the [update page](https://nhsuk-prototype-kit.azurewebsites.net/docs/how-tos/updating-the-kit) as most users doing upgrades won't see the changelog by default?

Reply:
Agree. Perhaps a more general mention, something like: check the frontend release notes for breaking changes that you may need to incorporate? To save having version specific stuff on the update page.

Reply:
Have added a link and mention in 3baa3468081e9d0a0dfd9bd080a98cf2e6cfaedb - is that enough?

Reply:
I think it should be more explicit that users will need to make changes to templates for the back links.